Automattic/wc-calypso-bridge
# Add store address preview mode to store setup

Fixes #93 

This PR adds in a preview mode (based on p58i-7kf-p2 ) for the address if an address is already available.

This PR is branched from https://github.com/Automattic/wc-calypso-bridge/pull/98 

### Screenshots
<img width="597" alt="screen shot 2018-10-26 at 5 03 05 pm" src="https://user-images.githubusercontent.com/10561050/47594469-0f718680-d941-11e8-8b02-faf3c42d7a71.png">

### Testing
1.  Check that you have an address already setup in your install (address line 1, city, state, zip, and country are required for the preview to be enabled).
2.  Visit `/wp-admin/admin.php?page=wc-setup`
3.  Verify that the address preview is there.
4.  Click "Edit" and verify that the address fields are displayed to allow editing of the address.
